mne/io/meas_info.py

```diff
@@ -54,6 +55,60 @@
 )
 
 
+def _get_valid_units():
```

Comment:
I'm not sure if we should have this here buried in our code or somewhere else with 'external'. If we had a dependency in BIDS I would rather like to do  `from bids import VALID_UNITS` which they export as a constant. Since thats not the case I would like it better to do `from mne.externals.bids import VALID_UNITS`

Reply:
These units are actually the international units ("SI" units): https://en.wikipedia.org/wiki/International_System_of_Units